packages/kit/src/views/Setting/pages/Tab/SettingListSubModal.tsx

```
@@ -0,0 +1,112 @@
+  return (
+    <Page scrollEnabled>
+      <Page.Header title={name} />
+      <Page.Body>
+        <YStack gap="$4" px="$4">
+          {isSearching ? (
+            <SearchView sections={searchResult} isSearching={isSearching} />
+          ) : (
+            configList?.map((item) => {
+              const list = Array.isArray(item) ? item.filter(Boolean) : [];
+              return list.length ? (
+                <TabSettingsSection>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)